src/components/tasks/card/index.tsx

```diff
@@ -131,6 +148,7 @@ const Card: FC<Props> = ({
                     new Date(`${event.target.value}`).getTime() / 1000;
                 toChange[changedProperty] = toTimeStamp;
             }
+            console.log(toChange);
```

Comment:
Could we remove this console log? I don't think we need console log when the code runs in production.